### ISSUE TITLE
Way to limit number of submitted job

### ISSUE THREAD

**Issue by bistace** [marbl/canu#433]
Hi,

I am trying to run Canu on a slurm cluster. My problem is that Canu tries to submit more than 300 jobs during the correction step but in my institure, a user can't submit more than 128 jobs at a time. I looked at the complete list of options available through the -options argument but couldn't find anything that could help me limit the number of jobs that are submitted, is there a way to do so ?

**Comment by skoren** [comment]
Canu submits a single array job with 300 elements, so this should only be counted as one "job" as far as the scheduler is concerned (it has a single jobid). The maximum that will be scheduled at a time is governed by the QoS of your cluster. Is your grid really dis-allowing a 300-array job?

There is no easy workaround for this, depending on which exact step in correction is submitting the jobs you'd have to adjust either the memory to be higher (```cormhapMemory```, try tripling your current value that should cut down the jobs) or the  ```corPartitions``` to set it to 100. Alternatively you could run on a single machine with ```useGrid=0```

**Comment by bistace** [comment]
Thanks for the fast reply! 

I talked with the sys admins and they told me we have a slightly modified slurm scheduler. One of the modification is that each job of an array has a unique batchid which makes them count as an independent job. 

The QoS that I need and use only allows up to 128 jobs per user so the submission was rejcted. I totally missed the option `corPartitions` when I wa searching for solutions. I set it to 127 and everything is now working as intended.
